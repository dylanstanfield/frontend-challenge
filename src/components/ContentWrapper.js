import { styled } from '@mui/material';

export const ContentWrapper = styled('div')(({ theme }) => `
  width: 100%;
  display: flex;
  flex-direction: column;
  gap: ${theme.spacing(4)};
`);