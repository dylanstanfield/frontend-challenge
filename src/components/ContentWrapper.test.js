import { render } from "@testing-library/react";

import { ContentWrapper } from "./ContentWrapper";

describe('ContentWrapper', () => {
  it('should match snapshot', () => {
    const { container } = render(<ContentWrapper>Hello World</ContentWrapper>);
    expect(container).toMatchSnapshot();
  });
});