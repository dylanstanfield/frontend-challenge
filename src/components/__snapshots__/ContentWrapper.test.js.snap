// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentWrapper should match snapshot 1`] = `
<div>
  <div
    class="css-pjtwp2"
  >
    Hello World
  </div>
</div>
`;
